### PR TITLE
Ensure match reboot popup for autoyast test on powerVM

### PR DESCRIPTION
**Pull request by lemon-suse**
- Related ticket: https://progress.opensuse.org/issues/174733
- Needles: N/A
- Verification run: https://openqa.suse.de/tests/16290943#step/installation/7
  https://openqa.suse.de/tests/16299882# regression test

### CODE REVIEW

File: tests/autoyast/installation.pm [hunk 1]
```diff
@@ -128,7 +128,7 @@ sub run {
     push @needles, 'autoyast-error' unless get_var('AUTOYAST_EXPECT_ERRORS');
     # Autoyast reboot automatically without confirmation, usually assert 'bios-boot' that is not existing on zVM
```
Comment (rfan1) [inline tests/autoyast/installation.pm:129]: THanks, could you please add comment here why automatic reboot without confirmation is disappear for sle15sp7?, do we have any jira ticket/bug report or some user requirements?

btw, howabout s390x? or other arrches, and other hypervisors :)

Comment (rfan1) [reply tests/autoyast/installation.pm:129]: > THanks, could you please add comment here why automatic reboot without confirmation is disappeared for sle15sp7?, do we have any jira ticket/bug report or some user requirements?
> 
> btw, how about s390x? or other arrches, and other hypervisors :) IMO they may get the popup windows as well

**Comment by rfan1** [comment]
One more question, is https://progress.opensuse.org/issues/174736 the right ticket refers to the failed job?

File: tests/autoyast/installation.pm (hunk 1 above)
Comment (lemon-suse) [reply tests/autoyast/installation.pm:129]: Thanks for review. In fact, the fix for https://bugzilla.suse.com/show_bug.cgi?id=1231522 add 'reboot' popup after installation on powerVM, user need click ok to reboot. For s390x, on latest build with the fix, it still be automatic reboot. For backport, developer is asking whether need backport to 15SP5 and 15SP6 in bug.
Comments added.

**Comment by lemon-suse** [comment]
> One more question, is https://progress.opensuse.org/issues/174736 the right ticket refers to the failed job?

Thanks, the ticket should be https://progress.opensuse.org/issues/174733

**Pull request merged**